RELEASING.md

```diff
@@ -15,3 +21,20 @@ gem push pkg/percy-capybara-X.XX.XX.gem
 * Announce the new release,
    making sure to say "thank you" to the contributors
    who helped shape this version!
+
+## Updating the vendorized version of percy-agent.js
```

Comment:
I think we should make this dynamic and pulled from percy-agent, so that we don't ever have to worry about version discrepancies between the percy-agent node version and this vendored JS.

Should be as simple as depending on `excon` gem and then:

```ruby
connection = Excon.new('http://localhost:5338/percy-agent.js', connect_timeout: 5)
percy_agent_js = connection.get.body

# And handle Excon::Error::Timeout safely?
```

Reply:
Done. (Used `Net::HTTP` library, though, since that's what I'm already using for interacting with the agent anyway.)